plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Payments.php

```diff
@@ -328,9 +333,14 @@ private function get_payments_settings_country(): string {
 	 */
 	private function get_payments_providers(): array {
 		try {
+			/**
+			 * The Payments Settings [page] service.
+			 *
+			 * @var SettingsPaymentsService $settings_payments_service
+			 */
 			$settings_payments_service = wc_get_container()->get( SettingsPaymentsService::class );
 
-			return $settings_payments_service->get_payment_providers( $settings_payments_service->get_country() );
+			return $settings_payments_service->get_payment_providers( $settings_payments_service->get_country(), false );
```

Comment:
Not setting `false` is an oversight from [a previous PR](https://github.com/woocommerce/woocommerce/pull/58580) due to the lack of IDE properly identifying the instance type.